pac4j-config/src/main/java/org/pac4j/config/builder/OAuthBuilder.java

```diff
@@ -121,4 +121,15 @@ public void tryCreateTwitterClient(final List<Client> clients) {
             clients.add(twitterClient);
         }
     }
+
+    public void tryCreateOauth2Client(final List<Client> clients) {
+        final String key = getProperty(OAUTH2_KEY);
+        final String secret = getProperty(OAUTH2_SECRET);
+        if (isNotBlank(key) && isNotBlank(secret)) {
+            final OAuth20Client oAuth20Client = new OAuth20Client();
```

Comment:
Did you test this change? I don't think it works.

You should use the `GenericOAuth20Client` and set more properties: `authUrl`, `tokenUrl`, ...